RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -628,20 +628,80 @@ def runTauID(self):
                     "VVTight": 0.9859
                 }
             }
-            file_name = 'RecoTauTag/TrainingFiles/data/DeepTauId/deepTau_2017v1_20L1024N_quantized.pb'
+            file_names = ['RecoTauTag/TrainingFiles/data/DeepTauId/deepTau_2017v1_20L1024N_quantized.pb']
             self.process.deepTau2017v1 = self.cms.EDProducer("DeepTauId",
                 electrons              = self.cms.InputTag('slimmedElectrons'),
                 muons                  = self.cms.InputTag('slimmedMuons'),
                 taus                   = self.cms.InputTag('slimmedTaus'),
-                graph_file             = self.cms.string(file_name),
-                mem_mapped             = self.cms.bool(False)
+                pfcands                = self.cms.InputTag('packedPFCandidates'),
+                vertices               = self.cms.InputTag('offlineSlimmedPrimaryVertices'),
+                rho                    = self.cms.InputTag('fixedGridRhoAll'),
+                graph_file             = self.cms.vstring(file_names),
+                mem_mapped             = self.cms.bool(False),
+                version                = self.cms.uint32(self.getDeepTauVersion(file_names[0])[1])
             )
 
             self.processDeepProducer('deepTau2017v1', tauIDSources, workingPoints_)
 
             self.process.rerunMvaIsolationTask.add(self.process.deepTau2017v1)
             self.process.rerunMvaIsolationSequence += self.process.deepTau2017v1
 
+        if "deepTau2017v2" in self.toKeep:
+            print ("Adding DeepTau IDs")
+
+            workingPoints_ = {
+                "e": {
+                    "VVVLoose": 0.0630386,
+                    "VVLoose": 0.1686942,
+                    "VLoose": 0.3628130,
+                    "Loose": 0.6815435,
+                    "Medium": 0.8847544,
+                    "Tight": 0.9675541,
+                    "VTight": 0.9859251,
+                    "VVTight": 0.9928449,
+                },
+                "mu": {
+                    "VLoose": 0.1058354,
+                    "Loose": 0.2158633,
+                    "Medium": 0.5551894,
+                    "Tight": 0.8754835,
+                },
+                "jet": {
+                    "VVVLoose": 0.2599605,
+                    "VVLoose": 0.4249705,
+                    "VLoose": 0.5983682,
+                    "Loose": 0.7848675,
+                    "Medium": 0.8834768,
+                    "Tight": 0.9308689,
+                    "VTight": 0.9573137,
+                    "VVTight": 0.9733927,
+                },
+            }
+            #file_names = ['RecoTauTag/TrainingFiles/data/DeepTauId/deepTau_2017v2p6_e6.pb']
```

Comment:
If this is outdated, it could be also removed from here, just to avoid confusion